test/UniswapV3Staker.integration.spec.ts

```diff
@@ -608,7 +608,7 @@ describe('UniswapV3Staker.integration', async () => {
         createIncentiveResult,
       })
 
-      expect(lp0Balance).to.eq(BN('1499999131944542011061'))
+      expect(lp0Balance).to.eq(BN('1499999131944544913825'))
```

Comment:
net changes:

`lp0Balance` increased by `2_902_764`
`lp1Balance` decreased by `345_271`
`lp2Balance` decreased by `207_082`

net increase (precision gain) of `2_350_411`